pgxn/neon_walredo/walredoproc.c

```diff
@@ -43,6 +43,10 @@
 
 #include "postgres.h"
 
+#ifdef HAVE_SHMEMPIPE
+#include "shmempipe.h"
```

Comment:
forgot to "git add" this?

Reply:
Yes, noticed around the same time as you commented it :)